profile/README.md

```diff
@@ -0,0 +1,38 @@
+# About Stoplight
+Stoplight is a collaborative API design platform that integrates seamlessly into your existing workflows to enable all stakeholders working with APIs to be more productive.
+
+<p align="center">
+  <img src="./assets/hero.svg" width="600" height="300" />
+</p>
+
+# Our Core Values
+At Stoplight, we ground ourselves around three guiding principles that unite us. We do our best work when we’re empowered, collaborative, and moving forward with positive intent.
+
+* ## <img src="./assets/owner.svg" width="20" height="20" /> Be An Owner
+  When we see an opportunity to make an impact, we seize it.
+
+* ## <img src="./assets/build.svg" width="20" height="20" /> Build Together
+  We trust in each other to add value and solve problems through teamwork.
+
+* ## <img src="./assets/mindfulness.svg" width="20" height="20" /> Practice Mindfulness
+  We seek to understand by listening, inquiring, and responding thoughtfully.
+
+# Mission
+
+We believe in a design-first approach to API development, so we created products that provide a means for design-first development. Developing good design-first practices will minimize future costs, speed up your time to market and lead to more consistent, higher quality microservice and IoT applications.
+
+<p align="center">
+  <img src="./assets/mission-team.png" width="600" height="300" />
+</p>
+
+# Social Good
+
+At Stoplight, we believe the positive impact we can make on the world isn’t limited to the API space. That’s why we work with organizations like , , and . Check out our  to learn how we offset the carbon footprint of every Stoplight employee and became a climate positive workforce.
+
+<div style="display: flex; justify-content: center;">
```

Comment:
Not sure if these will work or if the image source has to be in the repo as well

`![We offset our carbon footprint via Ecologi](https://api.ecologi.com/badges/cpw/5db9f5263b387200179a1d3d?black=true&landscape=true#gh-light-mode-only)
![We offset our carbon footprint via Ecologi](https://api.ecologi.com/badges/cpw/5db9f5263b387200179a1d3d?white=true&landscape=true#gh-dark-mode-only)`

or

`[![We offset our carbon footprint via Ecologi](https://api.ecologi.com/badges/cpw/5db9f5263b387200179a1d3d?black=true&landscape=true#gh-light-mode-only)](https://ecologi.com/stoplightinc?r=5db9f5263b387200179a1d3d)

[![We offset our carbon footprint via Ecologi](https://api.ecologi.com/badges/cpw/5db9f5263b387200179a1d3d?white=true&landscape=true#gh-dark-mode-only)](https://ecologi.com/stoplightinc?r=5db9f5263b387200179a1d3d)`